docs/setup/local-up.md

```diff
@@ -0,0 +1,27 @@
+### Deploy local Sedna clusters
+
+Once you have docker running you can create local Sedna clusters with:

```

Comment:
Add a description such as it uses kind for creating a k8s cluster, and a link for kind

Reply:
ok